pkg/reconciler/v1alpha1/autoscaling/kpa/kpa.go

```diff
@@ -232,10 +259,14 @@ func (c *Reconciler) reconcile(ctx context.Context, pa *pav1alpha1.PodAutoscaler
 	want, err := c.scaler.Scale(ctx, pa, decider.Status.DesiredScale)
 	if err != nil {
 		logger.Errorf("Error scaling target: %v", err)
-		return err
+		return 0, err
 	}
+	return want, nil
+}
+
+func (c *Reconciler) addressCount(ctx context.Context, pa *pav1alpha1.PodAutoscaler) (int, error) {
```

Comment:
See above, this function is no longer needed I think.

Reply:
Done.